openedx/core/djangoapps/content/course_overviews/utils.py

```diff
@@ -0,0 +1,15 @@
+"""
+Course Overview utils
+"""
+from django.conf import settings
+from lms.djangoapps.ccx.overrides import get_current_ccx
+
+
+def is_discussion_enabled(course_id):
```

Comment:
This doesn't belong in course_overviews since it's discussion forums specific.  See my comment above about finding a place in [django_comment_client](https://github.com/edx/edx-platform/tree/master/lms/djangoapps/django_comment_client).